### ISSUE TITLE
About Batch-Normalization

### ISSUE THREAD

**Issue by luyongxi** [tinghuiz/SfMLearner#76]
Thanks a lot for sharing your work! In the CVPR paper, it is stated that "batch normalization is used at all layers except for output layers". However, the network models in both this repo and the pytorch fork choose not to use batch norm. This seems to be quite important in training (from my experiments), could you provide some insights?

Thanks a lot!

**Comment by tinghuiz** [comment]
Please see previous discussion: https://github.com/tinghuiz/SfMLearner/issues/30.
